package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt",
   "description": "The JavaScript Task Runner",
-  "version": "1.0.0-rc1",
+  "version": "1.0.0-rc2",
   "author": "Grunt Development Team (http://gruntjs.com/development-team)",
   "homepage": "http://gruntjs.com/",
   "repository": "gruntjs/grunt",
```

Comment:
Sorry, just browsing from a mobile device here, but does NOT updating legacy util mean [this issue](https://github.com/gruntjs/grunt/issues/1463) doesn't get resolved/tested in rc2?